### PR TITLE
Sonarqube cleanup

### PR DESCRIPTION
**Pull request by feffi**
some code simplification
removed deprecated Throwables.propagate

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
 
-before_install:
-  - export MAVEN_OPTS=-Xmx1024m
-
-script:
-  - ./build/travis.sh
-
 jdk:
   - oraclejdk8
+  - oraclejdk9
```
Comment (jknack) [inline .travis.yml]: build don't work on Java9 or higher

File: jooby/src/main/java/org/jooby/Asset.java [hunk 2]
```diff
@@ -324,7 +326,13 @@ default String etag() {
     b.append("W/\"");
 
     BaseEncoding b64 = BaseEncoding.base64();
-    int lhash = resource().hashCode();
+    int lhash = 0;
+    try {
+      lhash = resource().toURI().hashCode();
```
Comment (jknack) [inline jooby/src/main/java/org/jooby/Asset.java]: why this change?

File: jooby/src/main/java/org/jooby/Asset.java [hunk 3]
```diff
@@ -324,7 +326,13 @@ default String etag() {
...
+    } catch (URISyntaxException e) {
+      throwIfUnchecked(e);
+      throw new RuntimeException(e);
```
Comment (jknack) [inline jooby/src/main/java/org/jooby/Asset.java]: URISyntaxException isn't an unchecked exception and I don't like to wrap exception as runtime. It is just noise

File: jooby/src/main/java/org/jooby/Err.java [hunk 4]
```diff
@@ -462,17 +462,17 @@ public int statusCode() {
    * @return A lightweight view of the err.
    */
   public Map<String, Object> toMap(boolean stacktrace) {
-    Status status = Status.valueOf(this.status);
+    Status statusOutput = Status.valueOf(this.status);
```
Comment (jknack) [inline jooby/src/main/java/org/jooby/Err.java]: why the name change? This is when sonar or similar tools don't make sense.

File: .travis.yml (hunk 1 above)
Comment (feffi) [reply .travis.yml]: Oh sorry, this change wasn't meant to be in this PR. Just disable it.

**Comment by feffi** [comment]
obsolete

**Comment by jknack** [comment]
Yea, I applied some of your changes manually to core (mainly).

Thanks
